### PR TITLE
Spanish docs [skip ci]

### PR DESCRIPTION
**Pull request by lucianobaraglia**

### CODE REVIEW

**Comment by lucianobaraglia** [comment]
@SilverFire, @cebe, @samdark sorry guys, but could someone accept this PR so I can keep working on the translation? I don't want to mess with the branches and merges...

File: docs/guide-es/start-looking-ahead.md [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-  en tu aplicación y lograr que sea más simple y rápido desarrollarla.
+  Yii puede hacer alarde de una librería de miles de extensiones contribuidas por usuarios, que pueden fácilmente conectadas a tu aplicación, haciendo que el desarrollo de la misma sea todavía más fácil y rápido.
 * Comunidad
-    - [Foro](http://www.yiiframework.com/forum/)
-    - [GitHub](https://github.com/yiisoft/yii2)
-    - [Facebook](https://www.facebook.com/groups/yiitalk/)
-    - [Twitter](https://twitter.com/yiiframework)
-    - [LinkedIn](https://www.linkedin.com/groups/yii-framework-1483367)
-
+    - Foro: <http://www.yiiframework.com/forum/>
```
Comment (klimov-paul) [inline docs/guide-es/start-looking-ahead.md]: Why links format has been chaged? Original one seems to be correct.

Comment (klimov-paul) [inline docs/guide-es/start-looking-ahead.md]: Ah, I see now. Everything is fine here.

**Comment by klimov-paul** [comment]
Thank you.

File: docs/guide-es/start-looking-ahead.md (hunk 1 above)
Comment (lucianobaraglia) [inline docs/guide-es/start-looking-ahead.md]: Now it's like that in the English Guide...I don't know really why...

**Comment by lucianobaraglia** [comment]
Thanks, @klimov-paul ! :smiley:

**Pull request merged**